src/components/Feature.tsx

```diff
@@ -3,7 +3,7 @@ import { storyblokEditable } from 'gatsby-source-storyblok';
 
 const Feature = ({ blok }) => (
   <div {...storyblokEditable(blok)} key={blok._uid}>
-    <h2>{blok.name}</h2>
+    <h2 className="su-text-gc-electric-blue su-bg-spirited">{blok.name}</h2>
```

Comment:
Added these funky TW classes to these temporary components

Reply:
We gonna rock down to Electric Avenue
And then we'll take it higher

Reply:
That's how we roll 🕺🏼 😄